### ISSUE TITLE
Cannot seem to get the "device_client_id_blob" and "device_private_key" files

### ISSUE THREAD

**Issue by guarded1** [sim0n00ps/OF-DL#212]
I've been going through the process to try and get OF DRM up and running.  I've gotten as far as downloading and using Android Studio but even following the tutorial that's linked from videohelp I haven't had any success getting those files.  Is there a simpler tutorial, or can someone send me those 2 files?  My Discord name is WandererOfWorlds#01

**Comment by sim0n00ps** [comment]
I believe I have sent you a friend request

**Comment by guarded1** [comment]
on discord?

**Comment by sim0n00ps** [comment]
> on discord?

Yes, the username you provided wasn't accepted so I just sent it to WandererOfWorlds with no # after it

**Comment by guarded1** [comment]
I didnt get the friend request.  I clicked and copied my user name here it is - wandererofworlds.

**Comment by guarded1** [comment]
let me try friend requesting you then?

**Comment by guarded1** [comment]
need your discord username

**Comment by guarded1** [comment]
friend request still hasn't come through.  I've been trying to use android studio and going back through the steps.  I get python installed and recognized and I get as far as the execution of this code as it is pictured on the videohelp forum tutorial 

adb.exe shell
su
mv /sdcard/frida-server_yourversion /data/local/tmp
chmod  +x /data/local/tmp/frida-server-yourversion
/data/local/tmp/frida-server-16.0.2-android-x86

obviously I replaced "yourversion" with the actual installed version, just in case anyone was ready to ask.  The 2nd line fails and gives me this error 

"
mv:  Need 2 arguments (see" mv  --help")

Each step prior to that in the tutorial was completed and everything went fine.  This current problem is where I got stuck

**Comment by MillerTimeJones** [comment]
same issue here i dont use discord but a friend said i can use his moneyberg247

**Comment by guarded1** [comment]
apparently my discord username includes the "." at the end for some reason.   wandererofworlds.

**Comment by sim0n00ps** [comment]
Should be sent now

**Comment by MillerTimeJones** [comment]
> Should be sent now

can i have one as well please?

thank you

**Comment by sim0n00ps** [comment]
> > Should be sent now
> 
> can i have one as well please?
> 
> thank you

Friend request sent

**Comment by MillerTimeJones** [comment]
> > > Should be sent now
> > 
> > 
> > can i have one as well please?
> > thank you
> 
> Friend request sent

added

**Comment by kauttoj** [comment]
FYI, when creating that virtual device in Android Studio, pick "Android 9.0" at API level 28 with x86 (not x86_64). Then, when running the dumper, make sure it uses option "--cdm-version '14.0.0'" that goes with Android 9.0. This was the only configuration I was able to get those two files out. My frida file was "frida-server-16.1.7-android-x86".

**Comment by jamesltyler9** [comment]
Can I be sent these as well? I have been struggling to get the Android program to work correctly.

**Comment by sim0n00ps** [comment]
> Can I be sent these as well? I have been struggling to get the Android program to work correctly. 
> 
> 

The files are no longer a mandatory requirement for DRM videos, the latest versions of OF-DL use cdrm-project as a failover if you don't have the 2 files

**Comment by Crazylocust** [comment]
Did you manage to get the files?
I have a ryzen cpu so android studio won't allow me to proceed.
